dspace/config/spring/api/scripts.xml

```diff
@@ -97,4 +97,9 @@
         <property name="dspaceRunnableClass" value="org.dspace.app.bulkaccesscontrol.BulkAccessControlCli"/>
     </bean>
 
+    <bean id="import-oaire-suggestions" class="org.dspace.app.suggestion.oaire.PublicationLoaderCliScriptConfiguration">
```

Comment:
This should be renamed `import-openaire-suggestions` so that it is similar to `import-openaire-events`.